src/components/Callout/Callout.tsx

```diff
@@ -17,8 +18,14 @@ export const Callout = ({
       variation="filled"
       colorTheme={warning ? 'warning' : 'info'}
       backgroundColor={backgroundColor}
+      hasIcon={false}
     >
-      <View>{children}</View>
+      <Flex>
+        <div role="img" aria-label="info" className="amplify-message__icon">
```

Comment:
Instead of wrapping this in an IMG role, I think you should be able to pass `aria-hidden="false"` to the Icon and pass it an appropriate aria-label.